packages/bot-web-ui/src/app/app.jsx

```diff
@@ -16,13 +16,21 @@ import RootStore from 'Stores';
 import GTM from 'Utils/gtm';
 import BotBuilder from 'Components/dashboard/bot-builder';
 import './app.scss';
+import { setColors } from '../../../bot-skeleton/src/scratch/hooks/colours';
```

Comment:
can we use alias to import from bot-skeleton?

Reply:
done ✅